### PR TITLE
Added support for more value types to filtering defaults

### PR DESCRIPTION
**Pull request by jfheins**
Some people use uints or other uncommon types in their data models. In my case, we have a number that can only be positive (usually between 0 and 12) and we made it an uint.

Ideally, I want to tell HC once to treat that uint as IntType.

To add some more value, I also included the rest o the basic types.

### CODE REVIEW

File: src/HotChocolate/Data/src/Data/Filters/Convention/Extensions/FilterConventionDescriptorExtensions.cs [hunk 1]
```diff
@@ -82,7 +82,11 @@ public static class FilterConventionDescriptorExtensions
                 .BindRuntimeType<DateTimeOffset?,
                     ComparableOperationFilterInputType<DateTimeOffset?>>()
                 .BindRuntimeType<TimeSpan?, ComparableOperationFilterInputType<TimeSpan?>>();
-
+                .BindRuntimeType<sbyte, ComparableOperationFilterInputType<sbyte>>()
```
Comment (PascalSenn) [inline src/HotChocolate/Data/src/Data/Filters/Convention/Extensions/FilterConventionDescriptorExtensions.cs:85]: Can you also add the nullables :) And also, they have a order, first the non nullable then the nullable
```csharp
 .BindRuntimeType<sbyte, ComparableOperationFilterInputType<sbyte>>()
 .BindRuntimeType<ushort, ComparableOperationFilterInputType<ushort>>()
 .BindRuntimeType<uint, ComparableOperationFilterInputType<uint>>()
 .BindRuntimeType<ulong, ComparableOperationFilterInputType<ulong>>()
....
 .BindRuntimeType<TimeSpan?, ComparableOperationFilterInputType<TimeSpan?>>();
 .BindRuntimeType<sbyte?, ComparableOperationFilterInputType<sbyte?>>()
 .BindRuntimeType<ushort?, ComparableOperationFilterInputType<ushort?>>()
 .BindRuntimeType<uint?, ComparableOperationFilterInputType<uint?>>()
 .BindRuntimeType<ulong?, ComparableOperationFilterInputType<ulong?>>()
```

**Review by PascalSenn — CHANGES_REQUESTED** [review]
Hi there

Thanks for the PR!

Small change, could you also add the nullable versions of the types?

File: src/HotChocolate/Data/src/Data/Filters/Convention/Extensions/FilterConventionDescriptorExtensions.cs (hunk 1 above)
Comment (jfheins) [reply src/HotChocolate/Data/src/Data/Filters/Convention/Extensions/FilterConventionDescriptorExtensions.cs]: Sure :-)
I also refactored the code a tiny bit to make it more obvious that all comparable types have also their nullable counterpart added. (But I can also revert that back if you don't like it)

**Review by PascalSenn — APPROVED** [review]
Sorry forgot about this one!

**Pull request merged**